integration-tests/api/package.json

```diff
@@ -29,12 +29,14 @@
   },
   "devDependencies": {
     "@opentelemetry/api": "^1.0.0",
+    "@opentelemetry/core": "^1.0.0",
     "@types/mocha": "10.0.7",
     "@types/node": "18.6.5",
     "codecov": "3.8.3",
     "cross-var": "1.1.0",
     "lerna": "6.6.2",
     "mocha": "10.0.0",
-    "nyc": "15.1.0"
+    "nyc": "15.1.0",
+    "ts-node": "^10.9.2"
```

Comment:
Do you know why this `ts-node` devDep was added?

Reply:
🤔  I wonder if we left that in when trying out some other integration tests. Doesn't look like we ended up using it for anything, should be okay to drop.

Reply:
I think the way we've set it up now will always require `ts-node` when we're using `mocha`, even if we're just feeding it JavaScript code (see `./.mocharc.yml`, which is the closes mocha config file).

So I think we'd actually need to add this `ts-node` dependency here.

Reply:
> Do you know why this `ts-node` devDep was added?

… I did last week. 

I removed `ts-node` from `package.json` and the root `package-lock.json`, rm'd root `./node_modules`, ran `npm run reset`, then this integration test `cd integration-tests/api && npm run test`, and the tests ran and passed. So … I can't explain the dev need for ts-node any more.

Reply:
Regarding changelog entries: should supply multiple entries to a CHANGELOG (one entry for every package touched in the closest parent CHANGELOG)? I would model each entry's content on [this past exports refactoring](https://github.com/open-telemetry/opentelemetry-js/pull/4785/files#diff-c9a82ab13ebd74152ffa3cc91afa636f11f19174034c8bc12990ae2a69c2c507R13).

Reply:
I think we should skip the changelog in the otlp-transformer package and the scripts directory, so we'd maybe want only in these:
- `.//CHANGELOG.md`
- `.//api/CHANGELOG.md`
- `.//experimental/CHANGELOG.md`

Reply:
Changelog entries added in fbe156fbd147fe9fba1a1477e0cb61c46de109b1. I opted to list each package individually in sub-bullets because I didn't see prior art for very long lists of updated packages. I'm happy to adjust that content—flatten to a single bullet with a comma-separated list, put the long list in `refactor(...):`, remove the list—however y'all see fit.

Reply:
> Update: I [removed ts-node from the devDeps](https://github.com/open-telemetry/opentelemetry-js/pull/4880/commits/9c0b671f724009a198c29624799c09d9d3a905da) and the test suite seems fine.

I understand the ts-node dep now. Marc's comment is correct. In https://github.com/open-telemetry/opentelemetry-js/pull/4840 we switched from using `ts-mocha` to just using `mocha` directly with a config file (https://github.com/open-telemetry/opentelemetry-js/blob/main/.mocharc.yml) that uses `ts-node`.  That means that technically any package.json file with a "mocha" devDep should also have a "ts-node" devDep. Because we are using npm workspaces with a shared top-level `node_modules/...` folder, **any single** package.json have a ts-node devDep that gets installed at the top-level saves us. And there are three:

```
% rg ts-node -g package.json
packages/opentelemetry-semantic-conventions/package.json
90:    "ts-node": "10.9.2",

experimental/examples/events/package.json
6:    "start": "ts-node index.ts"
18:    "ts-node": "^10.9.1"

experimental/examples/logs/package.json
6:    "start": "ts-node index.ts",
16:    "ts-node": "^10.9.1"
```

So, technically adding that devDep here was a good thing, but this is hardly the only transgressor. There are 3 package.json's with the "ts-node" dep and *44* with the "mocha" dep:

```
% rg '"mocha":' -g package.json | wc -l
      44
```